config/index.js

```diff
@@ -285,5 +288,46 @@ module.exports = {
     'wrap-regex': 'error',
     'yield-star-spacing': ['error', 'both'],
     'yoda': ['error', 'never'],
+
+    'import/default': 'error',
+    'import/dynamic-import-chunkname': 'off',
+    'import/export': 'error',
+    'import/exports-last': 'off',
+    'import/extensions': ['error', 'never', { 'json': 'always' }],
+    'import/first': 'error',
+    'import/group-exports': 'off',
+    'import/max-dependencies': 'off',
+    'import/named': 'error',
+    'import/namespace': 'error',
+    'import/newline-after-import': ['error', { 'count': 1 }],
+    'import/no-absolute-path': 'error',
+    'import/no-amd': 'off',
```

Comment:
We'd might as well turn this on, eh? We shouldn't have any AMD style stuff anywhere.

Reply:
Sure, yeah that makes sense